src/palimpzest/query/optimizer/optimizer.py

```diff
@@ -86,7 +86,7 @@ def __init__(
         self,
         policy: Policy,
         cost_model: CostModel,
-        no_cache: bool = False,
+        cache: bool = True,
```

Comment:
TODO: looks like this was another error that existed from before, we should set this to `cache: bool = False`